worker/src/main/scala/com/lucidchart/piezo/JobHistoryModel.scala

```diff
@@ -20,7 +19,16 @@ class JobHistoryModel(props: Properties) {
   val logger = LoggerFactory.getLogger(this.getClass)
   val connectionProvider = new ConnectionProvider(props)
 
-  def addJob(context: JobExecutionContext, success:Boolean): Unit = {
+  def addJob(
+    fireInstanceId: String,
+    jobName: String,
+    jobGroup: String,
+    triggerName: String,
```

Comment:
given that everywhere this is called we are using `trigger.getKey.getName` and `trigger.getKey.getGroup`,  I wonder if maybe instead of taking the name and group seperately, we should just take in whatever type `trigger.getKey` is.